app/models/post.rb

```diff
@@ -1,3 +1,20 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :user, foreign_key: 'author_id'
```

Comment:
[REQUIRED] Kindly change this line to this: 
`belongs_to :author, class_name: 'User'`
We don't have a foreign_key of author_id in user the foreign_key is in post

Reply:
Done! thanks!